spring-cloud-zookeeper-config/src/test/java/org/springframework/cloud/zookeeper/config/ZookeeperPropertySourceLocatorTests.java

```diff
@@ -16,6 +16,11 @@
 
 package org.springframework.cloud.zookeeper.config;
 
+import static org.hamcrest.Matchers.equalTo;
```

Comment:
Can you change the import in accordance with https://github.com/spring-cloud/spring-cloud-netflix/tree/master/eclipse ? If you're using Intellij Idea you can also import those there.